services/daq-nexus/values.yaml

```diff
@@ -0,0 +1,59 @@
+nexus-file-converter:
+  beamline: p46
+  template:
+    entry/:
+      NX_class@: NXentry
+      instrument/:
+        NX_class@: NXinstrument
+        name: p46
+
+  volumes:
+  - name: nexus-template
+    configMap:
+      name: nexus-template
+      items:
+      - key: template.yaml
+        path: template.yaml
+  - name: nexus-data
+    hostPath:
+      path: /dls/p46/data
+      type: DirectoryOrCreate
+
+  volumeMounts:
+  - name: nexus-data
+    mountPath: /dls/p46/data
+    mountPropagation: HostToContainer
+  - name: nexus-template
+    mountPath: /templates
+
+  application:
+    server:
+      port: 8084
+    spring:
+      profiles:
+        active: rabbitmq
+      rabbitmq:
+        host: daq-rabbitmq
+        username: p46
+    files:
+      output:
+        directory:
+          staticDirectory: /dls/p46/data/2024/cm37283-2/
+        nameTemplate: <DATA_SESSION>
+        devices:
+          types:
+            det: NX_DETECTOR
+            diff: NX_DETECTOR
+            panda1: NX_COLLECTION
+            panda2: NX_COLLECTION
+            chopper: NX_COLLECTION
+            sample_stage: NX_COLLECTION
+  containerEnvVars:
+  - name: SPRING_RABBITMQ_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: rabbitmq-password
+        key: rabbitmq-password
+  securityContext:
+    runAsUser: 37890 # p46-detector
+    runAsGroup: 37890
```

Comment:
:+1: I wasn't sure if you'd manage to get the right userids, but yep that's correct.